### ISSUE TITLE
Baseline pretrained model

### ISSUE THREAD

**Issue by ComputerMath** [TRI-ML/KP3D#10]
Hello, I'm looking for the resnet-baseline model which is only trained with coco dataset and got high performance in HPatches homography estimation (not uploaded model which is trained also with kitti.)

Can you upload the model above mentioned you have trained before?

Thank you in advance.

**Comment by RaresAmbrus** [comment]
Hmm I will try to find that one, but it may take some time. Maybe an option would be to try to train it with the [KP2D](https://github.com/TRI-ML/KP2D) codebase? I remember porting the resnet model to that codebase some time ago.

**Comment by ComputerMath** [comment]
Yes, you uploaded the resnet model in KP2D github site. (the model code and architecture is slightly different)
I think I can try, but i'm afraid that the trained model's performance is not achieved as the paper's as KP2D. (I uploaded this issue before in KP2D github issue and got some solution on my own)
I'm training the modified version your uploaded model, but i think the performance is not that great as paper.

It would be grateful if you find the model and upload it.

Thank you.
